src/app/views/home_page.js

```diff
@@ -0,0 +1,39 @@
+import View from 'ampersand-view';
+import mustache from 'mustache';
+import ExperimentRowView from './experiment_row_view';
+
+export default View.extend({
+  template: mustache.render('<section class="page"><ul class="experiments"></ul></section>'),
```

Comment:
We don't need it at the moment, but I'm planning to pull out the templates into actual separate files and use real mustaches and everything before too long.